ddcz/models/used/users.py

```diff
@@ -287,3 +290,27 @@ class UzivateleCekajici(models.Model):
     class Meta:
         db_table = "uzivatele_cekajici"
         unique_together = (("name_given", "name_family"),)
+
+    @property
+    def patron_name(self):
+        return UserProfile.objects.get(id=self.patron).nick
+
+    @property
+    def waiting_time(self):
+        time_now = int(time())
+        time_diff = time_now - self.date
+        return time_diff
+
+    @property
+    def expected_approval_time(self):
+        return self.date + APPROVAL_TIME
+
+    def patronize(self, patron_id):
+        try:
+            self.patron = patron_id
+            self.save()
+        except:
```

Comment:
I think this doesn't make sense—you are obfuscating the underlying error, raising something very generic that can't be catched later on and losing traceback.